### DISCUSSION TITLE
How to not create Sandbox\user\.... folders but relative Sandbox\drive\C\.... paths instead?

### DISCUSSION THREAD

**Discussion by parduz** [sandboxie-plus/Sandboxie#4371] [General]
Sorry for my english, i'm italian and i'll try to explain me:

I would like to not have the folders created in `user` (as example: `S:\MySandbox\user\current\AppData\Local`) and instead have it "expanded" to where they should be in the system drive (so, following my example, as `%LOCALAPPDATA%` is `C:\Users\Parduz\AppData\Local`  I'll have the `S:\MySandbox\drive\C\Users\Parduz\AppData\Local` folder ).

Is it possible?

**Comment by hg421** [comment]
Yes. Uncheck "Separate user folders" in Box settings -> File options:
![img](https://github.com/user-attachments/assets/a54550e1-421a-4049-9ebf-901ac56bd0c9)

**Comment by parduz** [comment]
Oh, thanks!
